### ISSUE TITLE
augmentation Documentation Tutorials

### ISSUE THREAD

**Issue by monkeycc** [ultralytics/ultralytics#1401]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar feature requests.

### Description

Data enhancement
augment Documentation Tutorials

The documentation tutorial does not have an example of how to use data enhancement

Want to add tutorials

Image augmentation for classification
Mask augmentation for segmentation
Bounding boxes augmentation for object detection

### Use case

```python
# A quick example
transform = [
    ["torchvision.Normalize", dict(mean=[0.5, 0.5, 0.5], std=[0.5, 0.5, 0.5])],
    ["torchvision.ToTensor", dict()]
]

#            T = [
#                A.Blur(p=0.01),
#                A.MedianBlur(p=0.01),
#                A.ToGray(p=0.01),
#               A.CLAHE(p=0.01),
#               A.RandomBrightnessContrast(p=0.0),
#                A.RandomGamma(p=0.0),
#               A.ImageCompression(quality_lower=75, p=0.0),]  # transforms

# Use the model
model.train(data="coco128.yaml",  train_transform=transform, epochs=3) 
```

### Additional

https://github.com/ultralytics/ultralytics/pull/956

### Are you willing to submit a PR?

- [x] Yes I'd like to help by submitting a PR!

**Comment by civerjia** [comment]
`model = YOLO()  # build a new model from scratch
model._new(cfg = "yolov8n_cfg.yaml")   
 model.train(data="data.yaml", epochs=100, batch = 32) 
`
copy everything in yolov8n.yaml
copy everything in args.yaml (once you trained the model, you can find it in runs/detect/train*/args.yaml)
create a new yolov8n_cfg.yaml file and set `augment: true`

**Comment by glenn-jocher** [comment]
@monkeycc if you'd like to help contribute a PR for these tutorials in the Docs that would be great. As your code shows we can use Torch transforms or Albumentation transforms. Albumentation transforms are documented in https://albumentations.ai/docs/api_reference/augmentations/transforms/

**Comment by glenn-jocher** [comment]
@AyushExel user request for improved Docs around augmentation.
